proxy.go

```diff
@@ -110,10 +111,39 @@ func main() {
 		Timeout: time.Second * 10,
 	}
 
-	http.HandleFunc(settingsString[LISTEN_PATH], uploadFile)
+	http.HandleFunc("/", proxyHandler)
 	http.ListenAndServe(":6743", nil)
 }
 
+func proxyHandler(w http.ResponseWriter, r *http.Request) {
+	if r.URL.Path == settingsString[LISTEN_PATH] && strings.HasPrefix(r.Header.Get("Content-Type"), "multipart/form-data") {
+		// Process request with a multipart/form-data
+		uploadFile(w, r)
+	} else {
+		// Forward the rest of requests
+		fmt.Println("Forwarding", r.Method, "request on", r.URL.Path)
+		forwardRequest(w, r)
+	}
+}
+
+func forwardRequest(w http.ResponseWriter, r *http.Request) {
```

Comment:
I feel like we should aim to have the uploadFile and forwardRequest to produce the same kind of forwarding. At the end of the day, the uploadFile is supposed to also forward the request, but with a resized file, which requires updating of the content-length header. Otherwise we should try to proxy the same thing and also transfer all headers. In uploadFile this is done with the copyHeader method, which I don't see here.

I'd recommend we put the proxying part into its own method and call it from both methods.